### ISSUE TITLE
Design decision: support multiple canvas and component namespaces on a single cluster

### ISSUE THREAD

**Issue by brian-burton** [tmforum-oda/oda-ca-docs#153]
## Design decision: support multiple canvas and component namespaces on a single cluster

We saw from the start that we might need to support multiple `canvas` and `components` namespaces on a single Kubernetes cluster, but although we've tried not to prevent that happening we also haven't designed in the ability to do it, consqeuently there are [a couple of things](https://github.com/tmforum-oda/oda-canvas/issues/207) that don't work properly.

I recommend that we adopt the principle that multiple `canvas` and `components` namespaces are supported so that we can formally begin to design for it.

**Comment by ferenc-hechler** [comment]
I agree, that component deployments should not be limited to only one namespace "components". 
One namespace per component maybe a good option to avoid any conflicts. 

But I do not see any advantages in having multiple canvases running in parallel in one cluster. 
The canvas deployment requests cluster wide resources (CRDs, WebHooks, ...) and multiple instances will get in conflict with each other instance. 
Even worse, if both instances are configured differently or have different versions.

**Comment by LesterThomas** [comment]
Added as [decision 14](https://github.com/tmforum-oda/oda-ca-docs/blob/master/Decision-Log/0014-multiple-component-namespaces.md)
